### DISCUSSION TITLE
Error: Using the "css" tag in runtime is not supported. Make sure you have set up the Babel plugin correctly.

### DISCUSSION THREAD

**Discussion by sunilbsharma** [Comcast/react-data-grid#3651] [General]
Hi,

I have integrated the react datagrid in my application and followed all features example. I am facing the Babel plugin issue.
Can you please help me with this?
My project is a TypeScript React on Windows 10.

Thanks!

**Comment by amanmahajan7** [comment]
This seems like a setup issue so I am moving it to a discussion. Can you also create a reproducible example
